src/setup.rs

```diff
@@ -204,3 +212,27 @@ async fn build_in_standalone_mode<Q: Executor + 'static>(
         .router(router)
         .schema_config_provider(schema_config_provider)
 }
+
+async fn create_static_topology_schema(
+    catalog_mgr: ManagerRef,
+    static_topology_config: StaticTopologyConfig,
+) {
+    let default_catalog = catalog_mgr
+        .catalog_by_name(catalog_mgr.default_catalog_name())
+        .expect("Fail to retrieve default catalog")
+        .expect("Default catalog doesn't exist");
+    for schema_shard_view in static_topology_config.schema_shards {
```

Comment:
Is it possible that duplicate schemas exists in the `schema_shards`? However, it is still fine if the `create_schema` won't fail for creating more than one same schema.